app/assets/stylesheets/screen.scss

```diff
@@ -706,3 +695,33 @@ div#keycut-help {
     padding-left: 45px;
   }
 }
+
+.fa-star {
+  color: #FFE000;
+}
+.fa-star-o {
+  color: #726904;
+}
+.fa-icon-overlay {
+  position: relative;
+}
+.fa-icon-overlay .fa {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+.fa-icon-overlay .fa:first-child {
+  position: initial;
+}
+.fa-comment {
+  color: white;
+}
+.fa-comment-o {
+  color: gray;
+}
+.fa-bug {
+  color: #C20B0B;
+}
+.fa-cog {
+  color: #5E5C5C;
+}
```

Comment:
ideia for this piece of code, that I see in some projects and really use the power of sass

```sass
.fa {
  &-star {
    color: #FFE000;
    
    &-o {
      color: #726904;
    }
  }

  &-icon-overlay {
    position: relative;
    
    .fa {
      position: absolute;
      top: 0;
      left: 0;
      
      &:first-child {
        position: initial;
      }
    }
  }
  
  &-comment {
    color: white;
    
    &-o {
      color: gray;
    }
  }

  &-bug {
    color: #C20B0B;
  }

  &-cog {
    color: #5E5C5C;
  }
}
```